docs/content/latest/explore/multi-region-deployments/row-level-geo-partitioning.md

```diff
@@ -330,35 +368,35 @@ txn_type      | debit
 created_at    | 2020-11-07 21:28:11.056236
 ```
 
-
-
 ## Step 5. Adding a new geographic location
 
 Assume that after a while, our fictitious Yuga Bank gets a lot of customers across the globe, and wants to offer the service to residents of Brazil, which also has data residency laws. Thanks to row-level geo-partitioning, this can be accomplished easily. We can simply add a new partition and pin it to the AWS South America (São Paulo) region `sa-east-1` as shown below.
 
-
-```
+```sql
 CREATE TABLE transactions_brazil 
     PARTITION OF transactions
       (user_id, account_id, geo_partition, account_type, 
        amount, txn_type, created_at,
        PRIMARY KEY (user_id HASH, account_id, geo_partition))
     FOR VALUES IN ('Brazil');
+```
 
+```sh
```

Comment:
Again, we won't use yb-admin commands anymore. Instead you would have to use the following commands here:
CREATE TABLESPACE sa_east_1_tablespace WITH (
  replica_placement='{"num_replicas": 3, "placement_blocks":
  [{"cloud":"aws","region":"sa-east-1","zone":"sa-east-1a","min_num_replicas":1},
  {"cloud":"aws","region":"sa-east-1","zone":"sa-east-1b","min_num_replicas":1},
  {"cloud":"aws","region":"sa-east-1","zone":"sa-east-1c","min_num_replicas":1}]}'
);


CREATE TABLE transactions_brazil 
     PARTITION OF transactions
       (user_id, account_id, geo_partition, account_type, 
        amount, txn_type, created_at,
        PRIMARY KEY (user_id HASH, account_id, geo_partition))
     FOR VALUES IN ('Brazil') TABLESPACE sa_east_1_tablespace;

Reply:
updated